packages/alfa-rules/test/sia-r5/rule.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import { test } from "@siteimprove/alfa-test";
+
+import { Document } from "@siteimprove/alfa-dom";
+
+import R5, { Outcomes } from "../../src/sia-r5/rule";
+
+import { evaluate } from "../common/evaluate";
+import { passed, failed, inapplicable } from "../common/outcome";
+
+test(`evaluate() passes an html element with lang attribute whose value is a valid primary language subtag`, async (t) => {
```

Comment:
```suggestion
test(`evaluate() passes a lang attribute with valid primary tag`, async (t) => {
```
We can be a bit more terse here, the full technically-precise language is needed on ACT rules but not really in the context of Alfa.
Plus, the rule (in Alfa) targeting the attribute rather than the element, we can match that in the description.